app/src/main/java/com/spencerpages/collections/KennedyHalfDollars.java

```diff
@@ -135,11 +212,6 @@ public int onCollectionDatabaseUpgrade(SQLiteDatabase db, CollectionListInfo col
                                            int oldVersion, int newVersion) {
 
         int total = 0;
-        if (oldVersion <= 2) {
-            // Need to add in 1968 - 1970 for Half Dollars unless it doesn't exist
-            // NOTE - We can't fix this, because adding will mess up the _id fields, which we use to do ordering
-            // We could make a new table and copy over all of the data, but that presents a lot of challenges
-        }
```

Comment:
Let's leave this in since it's unrelated to this PR - no functional different though